Source/Fuse.Models/FuseJS/Internal/Model.js

```diff
@@ -0,0 +1,489 @@
+var TreeObservable = require("FuseJS/TreeObservable")
+
+require("Polyfills/Window");
+require("./zone.min")
+require("./ZonePatches");
+
+var rootZone = Zone.current;
+
+function shouldEmitProperty(key) {
+	return !key.startsWith("__fuse_");
+}
+
+function isThenable(thing) {
+	return thing instanceof Object
+		&& typeof thing.then === "function";
+}
+
+function Model(initialState, stateInitializer)
+{
+	var stateToMeta = new Map();
+	var idToMeta = new Map();
+	var evaluatingDerivedProps = 0;
+	var idEnumerator = 0;
+	var store = this;
+
+	instrument(null, this, initialState, stateInitializer)
+
+	function instrument(parentMeta, node, state, stateInitializer)
+	{
+		if(stateInitializer instanceof Function) {
```

Comment:
The rest of the code does a space between `if` and `(` ...

Reply:
Actually, seems like we're pretty insonsistent... We should pick one and stick with it it (bonus points for adding it to 
[Documentation/CodingStyle.md](https://github.com/fusetools/fuselibs-public/blob/master/Documentation/CodingStyle.md) as well)